packages/@ourworldindata/grapher/src/stackedCharts/MarimekkoChart.tsx

```diff
@@ -1219,16 +1220,28 @@ export class MarimekkoChart
     20 labels relatively evenly spaced (in x domain space) and this function gives us 20 groups that
     are roughly of equal size and then we can pick the largest of each group */
     private static splitIntoEqualDomainSizeChunks(
+        items: Item[],
         candidates: LabelCandidate[],
         numChunks: number
     ): LabelCandidate[][] {
+        // we'll use this to validate a candidate against it's chart appearance validity
```

Comment:
I think it would be sensible to outline the difference between `items` and `candidates` here, something like:
> `candidates` contains labelling information for _all_ countries that are available in the chart. On the other hand, `items` contains the entities that are _currently_ being shown in the chart, i.e. for the currently selected year.